examples/01-simple-usage/recipe-type.ts

```diff
@@ -11,6 +11,9 @@ export class Recipe {
   @Field(type => Int)
   ratings: number[];
 
+  @Field({description: "Creation date field.", nullable: false})
```

Comment:
Date usage example is in the `typeorm` section with `@CreateDateColumn`. Simple example should be simple, description and depreciation examples has been added already.

Reply:
That's confusing, since there will be two main user bases on this lib, no-typeorm interested and typeorm interested ones.
Will there be an easy and complex example for each (no-typeorm and typeorm)?